data/Makefile.am

```diff
@@ -79,7 +79,9 @@ filetypes_dist = \
 	filedefs/filetypes.vhdl \
 	filedefs/filetypes.xml \
 	filedefs/filetypes.yaml \
-	filedefs/filetypes.zephir
+	filedefs/filetypes.zephir \
+	filedefs/filetypes.SCSS.conf \
+	filedefs/filetypes.LESS.conf
```

Comment:
These are sorted alphabetically in the list in the case-insensitive manner. Could you put them to the right places?